### DISCUSSION TITLE
Using Global POA Data for Simulation with pvlib

### DISCUSSION THREAD

**Discussion by yahayausman8** [pvlib/pvlib-python#2188] [Q&A] [answered]
Hello,

I have two datasets that I'd like to use as input for my simulation. The datasets are as follows:

1. DNI, DHI, and GHI: I successfully ran the simulation using these inputs, and it worked perfectly. Below is the code I used:

model_chain = ModelChain(system, site_location,
                 clearsky_model='ineichen', transposition_model='haydavies',
                 solar_position_method='nrel_numpy', airmass_model='kastenyoung1989',
                 dc_model=None, 
                 ac_model='sandia', 
                 aoi_model='ashrae',
                 spectral_model='no_loss',
                 temperature_model='sapm', losses_model='no_loss')

weather = pd.DataFrame([dat["dni"], dat["ghi"], dat["dhi"], dat["temp_air"], dat["wind_speed"]]).T
model_chain.run_model(weather)

2. Global POA: I also have Global Tilted Irradiance data from a pyranometer that is tilted the same as the PV module.

My question is: How can I perform the simulation using only the Global POA data?

Any guidance or suggestions would be greatly appreciated.

Thank you!

**Comment by AdamRJensen** [comment, marked as answer]
The easy option is to ignore aoi losses (then you don't need to know the split between poa_diffuse and poa_direct)

If you want to include aoi losses then you need to decompose the poa_global irradiance into poa_direct and poa_diffuse.  I'm aware of two algorithms in pvlib that are useful for this: [gti_dirint](https://pvlib-python.readthedocs.io/en/stable/reference/generated/pvlib.irradiance.gti_dirint.html) and [ghi_from_poa_diresse_2023](https://pvlib-python.readthedocs.io/en/stable/reference/generated/pvlib.irradiance.ghi_from_poa_driesse_2023.html). These two models will give you the ghi and ghi/dni/dhi respectively. You then have to transpose these components to your surface again but that will give you poa_diffuse and poa_direct. The sum of the derived poa_direct and poa_diffuse may not exactly match your original measured poa_global, which is why I personally would make some adjustments to the derived poa_diffuse/poa_direct to make sure they add up to poa_global.

**Reply by yahayausman8** [reply]
Thank you

**Comment by cwhanse** [comment]
Have a look at `modelchain.run_model_from_poa`. As @AdamRJensen says, you will need to have split the irradiance into `poa_direct` and `poa_diffuse`. Or, look at `modelchain.run_model_from_effective_irradiance` where you will need to assume that the effective irradiance is what is measured by the pyranometer. Effective irradiance is supposed to have been reduced by reflections and adjusted for spectral content. But maybe the pyranometer measurements are good enough for your work.

**Reply by yahayausman8** [reply]
Thank you for the explanation
